bootique-rabbitmq-client/src/test/java/io/bootique/rabbitmq/client/integration/RabbitMQNamingIT.java

```diff
@@ -0,0 +1,104 @@
+package io.bootique.rabbitmq.client.integration;
+
+import com.google.inject.Inject;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.GetResponse;
+import io.bootique.BQCoreModule;
+import io.bootique.BQRuntime;
+import io.bootique.Bootique;
+import io.bootique.rabbitmq.client.channel.ChannelFactory;
+import io.bootique.rabbitmq.client.connection.ConnectionFactory;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.WaitingConsumer;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.junit.Assert.assertEquals;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDOUT;
+
+/**
+ * Runs RabbitMQ and tests against real RMQ instance.
+ *
+ * @author Ibragimov Ruslan
+ * @since 0.24
+ */
+public class RabbitMQNamingIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RabbitMQNamingIT.class);
+
+    @ClassRule
+    public static GenericContainer rabbit = new GenericContainer("rabbitmq:3.6-alpine")
+            .withExposedPorts(5672);
+
+    @Test
+    public void test() throws IOException, TimeoutException {
+        LOGGER.info("Rabbit url: {}", rabbit.getContainerIpAddress() + ":" + rabbit.getMappedPort(5672));
+
+        BQRuntime runtime = Bootique
+                .app()
+                .args("--config=classpath:naming.yml")
+                .module((binder) -> {
+                    BQCoreModule.extend(binder)
+                            .setProperty(
+                                    "bq.rabbitmq.connections.bqConnection.port",
+                                    String.valueOf(rabbit.getMappedPort(5672))
+                            );
+                    binder.bind(RabbitMqUI.class);
+                })
+                .autoLoadModules()
+                .createRuntime();
+
+        waitForRabbit(rabbit);
+
+        runtime.getInstance(RabbitMqUI.class).init();
+    }
+
+    private static void waitForRabbit(GenericContainer container) throws TimeoutException {
```

Comment:
There is also declarative `.waitingFor(new LogMessageWaitStrategy().withRegEx(YOUR_REGEX))` on GenericContainer, just put it next to `withExposedPorts` :)